apps/flame-defi/app/features/evm-wallet/services/generic-contract-service.ts

```diff
@@ -47,7 +48,8 @@ export class GenericContractService {
         abi: this.abi,
         functionName: methodName,
         args,
-        value,
+        account: opts?.account,
```

Comment:
Is this necessary? Doesn't this get filled behind the scenes in based on them accepting the tx from their wallet?

Reply:
I thought this was necessary but I tested it out again and it is not. Will remove.